vendor/fof/drafts/js/src/forum/components/DraftsList.js

```diff
@@ -37,6 +37,35 @@ export default class DraftsList extends Component {
             <div className="NotificationList DraftsList">
                 <div className="NotificationList-header">
                     <h4 className="App-titleControl App-titleControl--text">{app.translator.trans('fof-drafts.forum.dropdown.title')}</h4>
+                    {Button.component({
+                                                      icon: 'fas fa-trash-alt',
+                                                      style: 'float: right; z-index: 20; margin-top: -25px',
+                                                      className: 'Button Button--icon Button--link draft--deleteAll draft--deleteAll',
+                                                      title: app.translator.trans('fof-drafts.forum.dropdown.delete_all_button'),
+                                                      onclick: (e) => {
+                                                        this.loading = false;
+                                                        if (!window.confirm(app.translator.trans('fof-drafts.forum.dropdown.delete_all_alert'))) return;
+                                                        var promise = [];
+                                                            drafts.forEach((draft) =>
+                                                                promise.push(
+                                                                    draft.delete().then(() => {
+                                                                        if (
+                                                                            app.composer.component &&
+                                                                            app.composer.component.draft.id() ===  drafts[element].id() &&
+                                                                            !app.composer.changed()
+                                                                        ) {
+                                                                            app.composer.hide();
+                                                                       }
+                                                                    })
+                                                                )
+                                                            );
+                                                            Promise.all(promise).then(() => {
```

Comment:
promise trả về rồi gọi redraw để update UI ->` m.redraw` phải trước` this.loading = false` chứ? @joseph280996

Reply:
em chi dua example vs viet tam 1 cai de em no test thoi chu luc do em chua test.